composer.json

```diff
@@ -20,6 +20,12 @@
             "tests/phpunit/util/"
         ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/OnTheGoSystems/collect.git"
+        }
+    ],
```

Comment:
@BruceOnTheGo please use the object form, as it makes easier to identify repositories and also allows manipulating them via CLI (which is not possible with the array form).

```json
"repositories": {
  "collect": {
    "type": "vcs",
    "url": "https://github.com/OnTheGoSystems/collect.git"
  }
}
```

The name of the repository can be anything you want, but "collect" (or "OnTheGoSystems/collect") is what makes more sense.

Reply:
@andreasciamanna Done